nixos/modules/misc/wordlist.nix

```
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+# Processes the list of paths
+# Returns either the sole path passed in if it's a singleton list.  Otherwise
+# creates a wordlist constructed from the various paths.
+processList = name: paths:
+  if builtins.length paths == 1
+  then
+    head paths
+  else
+    builtins.toString (pkgs.concatText name paths);
+in
+{
+  options = {
+    environment.wordlists = mkOption {
+      type = types.attrsOf (types.nonEmptyListOf types.path);
+
+      default = {
+        WORDLIST = [ "${pkgs.scowl}/share/dict/words.txt" ];
+      };
+
+      defaultText = literalExpression ''{
+        WORDLIST = [ "''${pkgs.scowl}/share/dict/words.txt" ];
+      };
+      '';
```

```suggestion
      defaultText = literalExpression ''
        {
          WORDLIST = [ "''${pkgs.scowl}/share/dict/words.txt" ];
        }
      '';
```
leading common whitespace is removed in `''`-quoted strings, but this one has none. the string as written looks like this when printed:
```
{
        WORDLIST = [ "''${pkgs.scowl}/share/dict/words.txt" ];
      };
```
which isn't usually the intention. the same goes for other `''` strings in this module.

here the trailing `;` is also misplaced here since it's not a part of the expression, rather it's part of an attribute binding.

---

oh dang, thanks for the catch, I thought I had fixed all of these errors after the last time you pointed this out.